hrl/agent/dsc/MBOptionClass.py

```diff
@@ -128,8 +128,9 @@ def get_training_phase(self):
         return "initiation_done"
 
     def extract_features_for_initiation_classifier(self, state):
```

Comment:
Looks like we have this function both in the MBOptionClass and the GoalDirectedEnvWrapper. Can we limit to defining it in one place?

Reply:
good point!

Reply:
```python
if "push" in self.mdp.unwrapped.spec.id:
            return features[:4]
```
is this ever used?

Reply:
Rather than having env specific code in the Option class, I would rather push that responsibility to the Env